module/input/Camera/src/Camera.cpp

```diff
@@ -47,7 +50,7 @@ namespace input {
              ************************************************************************/
             if (it == cameras.end()) {
                 // Strip the .yaml off the name of the file to get the name of the camera
-                std::string name = ::basename(config.path.substr(0, config.path.find_last_of('.')).c_str());
+                std::string name = ::basename(config.fileName.substr(0, config.fileName.find_last_of('.')).c_str());
```

Comment:
`snake_case`?

Reply:
Not my fault, the Horus one is path, your configuration is fileName

Reply:
You should update it then

Reply:
Fine, fix it later then